qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -866,7 +866,7 @@ def __del__(self):
                 'Memory should have been released before buffer was deleted.')
 
 
-class AcquisitionController(Instrument):
+class AcquisitionInterface:
     """
     This class represents all choices that the end-user has to make regarding
     the data-acquisition. this class should be subclassed to program these
```

Comment:
shall `acquisitioncontroller` be substrituded with `acquisitioninterface` in this docstring? (see below this line)

Reply:
Oh yes, I forgot the doc string, thanks!